guest/filter/filter.go

```diff
@@ -115,3 +116,57 @@ func (n *nodeInfo) lazyNode() proto.Node {
 	n.node = &internalproto.Node{Msg: &msg}
 	return n.node
 }
+
```

Comment:
nit: I am not sure PodInfo should be defined in this file.

Reply:
PodInfo is only being used in PreFilterExtentions, _for now_.
@chansuke You can move it to the same place where _addpod and _removepod is defined.